uninit_filter.py

```diff
@@ -26,10 +29,25 @@ def find_uninit_classes(coverage_file):
     return uninit_dict
 
 
+def output(uninit_dict, output_file):
+    if output_file is not None:
```

Comment:
I thought more about this output file thing, I think its probably easier to redirect stdout to a file if you want on the command line.  So instead of:
`python script.py -o output.txt`
you would write
`python script.py > output.txt`

This simplies your code a good bit because now print writes to the file and you don't need all these if statements or a context manager for the output file.